app/models/user.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = function(sequelize, DataTypes) {
```

Comment:
Here cant you use arrow functions instead of `function` ? Theres not difference in functionality in this case, its just to mantain the same coding style throughout the whole project.
The only case in which you would NEED to use `function` would be if there is a property in `this` that a library or something required you to use but albeit its not really recommended as scope in `function` declarations on `this` does not extend to closures.
If this isnt clear I can clarify further.

Reply:
Cool ! Since this was just autogenerated code I left it as it is (I just change some style things) but you are right